projects/plugins/jetpack/extensions/blocks/rating-star/edit.js

```diff
@@ -87,18 +87,18 @@ export default Symbol =>
 							max={ 10 }
 							__nextHasNoMarginBottom={ true }
 						/>
-						<PanelColorSettings
-							title={ __( 'Color Settings', 'jetpack' ) }
-							initialOpen
-							colorSettings={ [
-								{
-									value: color,
-									onChange: setNewColor,
-									label: __( 'Color', 'jetpack' ),
-								},
-							] }
-						/>
 					</PanelBody>
+					<PanelColorSettings
+						title={ __( 'Color', 'jetpack' ) }
```

Comment:
While the duplication in `Color` label is unfortunate, I still think this change removing "settings" is an improvement.